pep-0670.rst

```diff
@@ -2,11 +2,16 @@ PEP: 670
 Title: Convert macros to functions in the Python C API
 Author: Erlend Egeberg Aasland <erlend.aasland@protonmail.com>,
         Victor Stinner <vstinner@python.org>
-Status: Draft
+Status: Accepted
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 19-Oct-2021
 Python-Version: 3.11
+Resolution: https://mail.python.org/archives/list/python-dev@python.org/thread/QQFCJ7LR36RUZSC3WI6WZZMQVQ3ZI4MS/
+Post-History: `15-Oct-2021 <https://github.com/python/cpython/issues/89653>`__,
+              `20 Oct 2021 <https://mail.python.org/archives/list/python-dev@python.org/thread/2GN646CGWGTO6ZHHU7JTA5XWDF4ULM77/>`__,
+              `8 Feb 2022 <https://mail.python.org/archives/list/python-dev@python.org/message/IJ3IBVY3JDPROKX55YNDT6XZTVTTPGOP/>`__,
+              `22 Feb 2022 <https://mail.python.org/archives/list/python-dev@python.org/thread/VM6I3UHVMME6QRSUOYLK6N2OZHP454W6/>`__
```

Comment:
```suggestion
Post-History: `20-Oct-2021 <https://mail.python.org/archives/list/python-dev@python.org/thread/2GN646CGWGTO6ZHHU7JTA5XWDF4ULM77/>`__,
              `08-Feb-2022 <https://mail.python.org/archives/list/python-dev@python.org/message/IJ3IBVY3JDPROKX55YNDT6XZTVTTPGOP/>`__,
              `22-Feb-2022 <https://mail.python.org/archives/list/python-dev@python.org/thread/VM6I3UHVMME6QRSUOYLK6N2OZHP454W6/>`__
Resolution: https://mail.python.org/archives/list/python-dev@python.org/thread/QQFCJ7LR36RUZSC3WI6WZZMQVQ3ZI4MS/
```

A few things this fixes:

* Resolution always goes before post history
* Its only needed to link mailing list, discourse, etc. posts of the PEP itself here; bug reports can be linked inline in the text as applicable (as I assume they already have)
* A few of the date formats were inconsistent

Just to note, #2484 updates the linters (which you can also run automatically or on-demand locally, if you wish) to automatically flag each of these issues and give specific and immediate feedback on them.